internal/glinstance/host_test.go

```diff
@@ -242,3 +242,55 @@ func Test(t *testing.T) {
 		})
 	}
 }
+
+func Test_GraphQLEndpoint(t *testing.T) {
+	testCases := []struct {
+		name     string
+		protocol string
+		hostname string
+		output   string
+	}{
+		{
+			name:     "OfficialInstance/https",
+			protocol: "https",
+			hostname: "gitlab.com",
+			output:   "https://gitlab.com/api/graphql/",
+		},
+		{
+			name:     "OfficialInstance/http",
+			protocol: "http",
+			hostname: "gitlab.com",
+			output:   "http://gitlab.com/api/graphql/",
```

Comment:
with the official GitLab.com instance, the function always returns an endpoint with `https` protocol.

Here's the function:
```go
func GraphQLEndpoint(hostname, protocol string) string {
	if protocol == "" {
		protocol = "https"
	}
	if IsSelfHosted(hostname) {
		return fmt.Sprintf("%s://%s/api/graphql/", protocol, hostname)
	}
	return "https://gitlab.com/api/graphql/"
}
```